examples/action/ETH/ETH.xml

```diff
@@ -0,0 +1,243 @@
+                </sequence>
+            </type>
+        </namedType>
+    </asnx:module>
+
+    <ts:label>
+        <ts:string xml:lang="en">ETH</ts:string>
+    </ts:label>
+
+    <ts:contract name="balance">
+        <ts:address network="1">0xb8c9fc923e67da6d7979bf2c290bd1df9c512bfa</ts:address>
+    </ts:contract>
+    <ts:contract name="aETH" interface="erc20">
+        <ts:address network="1">0x3a3a65aab0dd2a17e3f1947ba16138cd37d08c04</ts:address>
+    </ts:contract>
+    <ts:contract name="ETH" interface="erc20">
+        <ts:address network="1">0x0000000000000000000000000000000000000000</ts:address>
+    </ts:contract>
+    <ts:contract name="LendingPoolProvider">
+        <ts:address network="1">0x24a42fD28C976A61Df5D00D0599C34c4f90748c8</ts:address>
+    </ts:contract>
+    <ts:contract name="AAVELendingPool">
+        <ts:address network="1">0x398eC7346DcD622eDc5ae82352F02bE94C62d119</ts:address>
+    </ts:contract>
+    <ts:contract name="AAVELendingPoolCore">
+        <ts:address network="1">0x3dfd23A6c5E8BbcFc9581d2E864a68feb6a076d3</ts:address>
+    </ts:contract>
+
+    <ts:origins>
+        <ts:ethereum contract="ETH"/>
+    </ts:origins>
+
+    <ts:cards>
+        <ts:card type="action">
+            <ts:label>
+                <ts:string xml:lang="en">Deposit</ts:string>
+                <ts:string xml:lang="zh">存款</ts:string>
+            </ts:label>
+            <ts:attribute name="mintAmount">
+                <ts:type><ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax></ts:type>
+                <ts:label>
+                    <ts:string xml:lang="en">Amount to Deposit</ts:string>
+                    <ts:string xml:lang="zh">金额</ts:string>
+                </ts:label>
+                <ts:origins>
+                    <!-- e18 is a hard coded multiplier.
+                    rationale for hardcoding: avoiding over-design  -->
+                    <ts:user-entry as="e18"/>
+                </ts:origins>
+            </ts:attribute>
+            <ts:transaction>
+                <ethereum:transaction function="deposit" contract="LendingPoolProvider" as="uint">
+                    <ts:data>
+                        <ts:address>0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE</ts:address>
+                        <ts:uint256 ref="mintAmount"/>
+                        <ts:uint256>1101</ts:uint256> <!-- placeholder referral -->
+                    </ts:data>
+                </ethereum:transaction>
+            </ts:transaction>
+            <ts:view xml:lang="en">
+                <xhtml:style type="text/css">&style;</xhtml:style>
+                <xhtml:script type="text/javascript">&deposit.en;</xhtml:script>
+            </ts:view>
+        </ts:card>
+
+        <ts:card type="activity" name="aETHMinted">
+            <ts:origins>
+                <ethereum:event type="MintOnDeposit" filter="ownerAddress=${_fromMinter}" contract="aETH"/>
```

Comment:
Unfortunately, you'll have to switch the filters around so it becomes:

```
filter="_fromMinter=${ownerAddress}"
```

I'll check after the train leaves if we can also support the original way you wrote it.